Common/Util/ResolutionTimeSpan.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace QuantConnect.Util
+{
+    /// <summary>
+    /// This class allows implicit conversions between Resolution and TimeSpan
+    /// </summary>
+    public class ResolutionTimeSpan
+    {
+        public ResolutionTimeSpan(TimeSpan timeSpan) { val = timeSpan; }
```

Comment:
Please add XML documentation and format this function according to the prevailing project's code style.